src/magic_folder/config.py

```diff
@@ -1161,24 +1174,25 @@ def get_all_current_snapshot_pathstates(self, cursor):
         'current_snapshot' table.
 
         :returns Iterable[(unicode, PathState)]: an iterable of
-            2-tuples of (name, PathState instance), one for each file
-            (ordered by timestamp)
+            3-tuples of (name, PathState instance, last-update), one for each file
+            (ordered by last-updated timestamp)
         """
         cursor.execute(
             """
             SELECT
-                name, mtime_ns, ctime_ns, size
+                name, mtime_ns, ctime_ns, size, last_updated_ns
             FROM
                 current_snapshots
             ORDER BY
-                mtime_ns DESC
+                last_updated_ns DESC
             """,
         )
 
         return [
             (
                 row[0],  # name
                 PathState(mtime_ns=row[1], ctime_ns=row[2], size=row[3]),
+                ns_to_seconds(row[4]),  # last_updated_ns
```

Comment:
Although the fact that we don't expose anything other than seconds outside the database leans me towards storing seconds in the db.